src/DssSpell.t.sol

```diff
@@ -1268,31 +1478,90 @@ contract DssSpellTest is DSTest, DSMath {
                 // Convert BP to system expected value
-                uint256 normalizedTestBeg = (values.collaterals[ilk].beg + 10000)  * 10**14;
+                uint256 normalizedTestBeg = (values.collaterals[ilk].flip_beg + 10000)  * 10**14;
                 assertEq(uint256(flip.beg()), normalizedTestBeg);
                 assertTrue(flip.beg() >= WAD && flip.beg() <= 110 * WAD / 100); // gte 0% and lte 10%
-                assertEq(uint256(flip.ttl()), values.collaterals[ilk].ttl);
+                assertEq(uint256(flip.ttl()), values.collaterals[ilk].flip_ttl);
                 assertTrue(flip.ttl() >= 600 && flip.ttl() < 10 hours);         // gt eq 10 minutes and lt 10 hours
-                assertEq(uint256(flip.tau()), values.collaterals[ilk].tau);
+                assertEq(uint256(flip.tau()), values.collaterals[ilk].flip_tau);
                 assertTrue(flip.tau() >= 600 && flip.tau() <= 3 days);          // gt eq 10 minutes and lt eq 3 days
 
                 assertEq(flip.wards(address(flipMom)), values.collaterals[ilk].flipper_mom);
 
-                assertEq(flip.wards(address(cat)), values.collaterals[ilk].liquidations);  // liquidations == 1 => on
-                checkWards(address(flip), bytes32ToStr(ilk));
+                assertEq(flip.wards(address(cat)), values.collaterals[ilk].liqOn ? 1 : 0);
                 assertEq(flip.wards(address(pauseProxy)), 1); // Check pause_proxy ward
                 }
+            }
+            if (values.collaterals[ilk].liqType == "clip") {
+                {
+                assertEq(reg.class(ilk), 1);
+                (bool ok, bytes memory val) = reg.xlip(ilk).call(abi.encodeWithSignature("dog()"));
+                assertTrue(ok);
+                assertEq(abi.decode(val, (address)), address(dog));
+                }
+                {
+                (, uint256 chop, uint256 hole,) = dog.ilks(ilk);
+                // Convert BP to system expected value
+                uint256 normalizedTestChop = (values.collaterals[ilk].chop * 10**14) + WAD;
+                assertEq(chop, normalizedTestChop);
+                // make sure chop is less than 100%
+                assertTrue(chop >= WAD && chop < 2 * WAD);   // penalty gt eq 0% and lt 100%
+
+                // Convert whole Dai units to expected RAD
+                uint256 normalizedTesthole = values.collaterals[ilk].dog_hole * RAD;
+                assertEq(hole, normalizedTesthole);
+                assertTrue(hole >= RAD && hole < 20 * MILLION * RAD);
+                }
+                (address clipper,,,) = dog.ilks(ilk);
+                ClipAbstract clip = ClipAbstract(clipper);
+                {
+                // Convert BP to system expected value
+                uint256 normalizedTestBuf = (values.collaterals[ilk].clip_buf + 10000)  * 10**23;
+                assertEq(uint256(clip.buf()), normalizedTestBuf);
+                assertTrue(clip.buf() >= RAY && clip.buf() <= 2 * RAY); // gte 0% and lte 100%
```

Comment:
Comment fix. Should be: `// gte 100% and lte 200%`

Reply:
Fixed